atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -236,8 +235,7 @@ public static SerializableTransactionManager create(
                 atlasFactory::getTimestampService,
                 atlasFactory.getTimestampStoreInvalidator(),
                 userAgent);
-        KeyValueService kvs = NamespacedKeyValueServices.wrapWithStaticNamespaceMappingKvs(rawKvs,
-                config.initializeAsync());
+        KeyValueService kvs = rawKvs;
```

Comment:
This line is now unnecessary - simply `KeyValueService kvs = ProfilingKeyValueService.create(rawKvs, ...)` will be fine.